chef/data_bags/crowbar/template-crowbar.json

```diff
@@ -20,7 +20,7 @@
       },
       "simple_proposal_ui": true,
       "upgrade": {
-        "db_dump_location" : "/var/lib/pgsql/db-before-cloud6.dump"
+        "db_dump_location" : "/var/lib/pgsql/upgrade-openstack-database.dump"
```

Comment:
This name doesn't really make sense. Would be better to use something like openstack-db-pre-upgrade.dump